src/lib/components/ResultsMultiLayout/ResultsMultiLayout.js

```diff
@@ -38,15 +38,19 @@ ResultsMultiLayout.defaultProps = {
 const Element = ({ layout, overridableId }) => {
   const { buildUID } = useContext(AppContext);
 
+  const handleMathjaxCallback = useCallback(() => {
+    window.MathJax?.typeset();
+  }, []);
+
   return (
     <Overridable
       id={buildUID("ResultsMultiLayout.element", overridableId)}
       layout={layout}
     >
       {layout === "list" ? (
-        <ResultsList overridableId={overridableId} />
+        <ResultsList overridableId={overridableId} mathjaxCallback={handleMathjaxCallback}/>
```

Comment:
isn't it a bit too specific? Shouldn't we pass a callback (as a name) and use it for mathjax?

Reply:
I agree with @kpsherva , I missed that :) Probably we should rename it to something `onRender`